app/graphql/types/address_type.rb

```diff
@@ -4,12 +4,26 @@ class AddressType < Types::BaseObject
     description 'An address representing a physical location'
 
     # field :id, ID, null: false
-    field :street_address, String, method: :full_street_address
-    field :address_locality, String
-    field :address_region, String
-    field :address_country, String, method: :country_code
-    field :postal_code, String, method: :postcode
-    field :neighbourhood, NeighbourhoodType
+    field :street_address, String, 
+      method: :full_street_address,
+      description: 'The first few lines of the address like: 123 Road, Eastleigh, Southampton'
+
+    field :address_locality, String,
+      description: 'The nieghbourhood name of this address (see neighbourhood type)'
+
+    field :address_region, String,
+      description: 'The neighbourhood region of this address (see neighbourhood type)'
+
+    field :address_country, String, 
+      method: :country_code,
+      description: 'The country of this event (at the moment is only UK)'
+
+    field :postal_code, String, 
+      method: :postcode,
+      description: '(UK) Postcode of address'
```

Comment:
i think this description is tautological lol but 'A UK postcode' might be neater